dbt_platform_helper/providers/config_validator.py

```diff
@@ -207,20 +207,12 @@ def validate_database_copy_section(self, config):
                     )
 
                 if from_account != to_account:
-                    if "from_account" not in section:
-                        errors.append(
-                            f"Environments '{from_env}' and '{to_env}' are in different AWS accounts. The 'from_account' parameter must be present."
-                        )
-                    elif section["from_account"] != from_account:
+                    if "from_account" in section and section["from_account"] != from_account:
```

Comment:
Perhaps worth a comment here to note that this is now deprecated so we know to remove it once tpm is merge in.